medusa/scene_exceptions.py

```diff
@@ -218,11 +193,23 @@ def update_scene_exceptions(indexer_id, indexer, scene_exceptions, season=-1):
     # TODO: make sure we add indexer, when the global exceptionsCache var has been changed.
     if indexer_id in exceptions_cache:
         exceptions_cache[indexer_id] = {}
-        exceptions_cache[indexer_id][season] = [se.decode('utf-8') for se in scene_exceptions]
 
-    for cur_exception in [se.decode('utf-8') for se in scene_exceptions]:
-        cache_db_con.action(b'INSERT INTO scene_exceptions (indexer_id, show_name, season, indexer) VALUES (?,?,?,?)',
-                            [indexer_id, cur_exception, season, indexer])
+    decoded_scene_exceptions = [se.decode('utf-8') for se in scene_exceptions]
+    for cur_exception in decoded_scene_exceptions:
+        if indexer_id not in exceptions_cache:
+            exceptions_cache[indexer_id] = {}
```

Comment:
Here we create it if it doesn't exists...
I suggest we just create it once without any checks before the loop.
If it exists, it will be cleared, if it doesn't, it will be created.